### ISSUE TITLE
result when run test.py

### ISSUE THREAD

**Issue by cuongngm** [wenwenyu/PICK-pytorch#68]
Tks for your repo, the code is very clean. I trained your model with my dataset, the loss is converged good. But when predicted, a few the value of item['text'] was different format with the value of file .tsv. Example: "ÁO MOA TT CÀI CÚC" vs "ÁO MƯA TT CÀI CÚC",..vv Can you help me explain?

**Comment by cuongngm** [comment]
i solved problem by created my vocab in my languague and retrained model

**Comment by babyhockey** [comment]
Hi Could you explain how you created the vocab in your language? Did you list all possible words (or characters) in keys.txt? Looking forward to your reply. Thanks!

**Comment by cuongngm** [comment]
that's right

**Comment by babyhockey** [comment]
Thank you!

**Comment by khangnguyenhuu** [comment]
hi there, can u give me an example in vietnamese vocab :( thanks a lot

**Comment by cuongngm** [comment]
give me your email and i will send for you example of Vietnamese vocab

**Comment by khangnguyenhuu** [comment]
> give me your email and i will send for you example of Vietnamese vocab
nskhang1@gmail.com, thanks a lot

**Comment by htdung167** [comment]
@cuongngm I'm having the same problem. Can you give me Vietnamese vocab?
 I am using vocab: 
#ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz| 0123456789ĂÂÊÔƠƯÁẮẤÉẾÍÓỐỚÚỨÝÀẰẦÈỀÌÒỒỜÙỪỲẢẲẨĐẺỂỈỎỔỞỦỬỶÃẴẪẼỄĨÕỖỠŨỮỸẠẶẬẸỆỊỌỘỢỤỰỴăâêôơưáắấéếíóốớúứýàằầèềìòồờùừỳảẳẩđẻểỉỏổởủửỷãẵẫẽễĩõỗỡũữỹạặậẹệịọộợụựỵ\'*:,@.-(#%")/~!^&_+={}[]\;<>?※”$€£¥₫°²™ā–
aAàÀảẢãÃáÁạẠăĂằẰẳẲẵẴắẮặẶâÂầẦẩẨẫẪấẤậẬbBcCdDđĐeEèÈẻẺẽẼéÉẹẸêÊềỀểỂễỄếẾệỆfFgGhHiIìÌỉỈĩĨíÍịỊjJkKlLmMnNoOòÒỏỎõÕóÓọỌôÔồỒổỔỗỖốỐộỘơƠờỜởỞỡỠớỚợỢpPqQrRsStTuUùÙủỦũŨúÚụỤưƯừỪửỬữỮứỨựỰvVwWxXyYỳỲỷỶỹỸýÝỵỴzZ0123456789!"#$%&''()*+,-./:;<=>?@[\]^_`{|}~
